lib/vtls/schannel.c

```diff
@@ -1716,4 +1903,69 @@ static CURLcode verify_certificate(struct connectdata *conn, int sockindex)
 }
 #endif /* _WIN32_WCE */
 
+static void Curl_schannel_checksum(const unsigned char *input,
```

Comment:
I would advise to have a return value that tells the user of this function if the checksum failed or not.